src/wallet/rpcwallet.cpp

```diff
@@ -4185,7 +4188,7 @@ UniValue z_mergetoaddress(const UniValue& params, bool fHelp)
             "  \"mergingTransparentValue\": xxx      (numeric) Value of UTXOs being merged.\n"
             "  \"mergingNotes\": xxx                 (numeric) Number of notes being merged.\n"
             "  \"mergingShieldedValue\": xxx         (numeric) Value of notes being merged.\n"
-            "  \"opid\": xxx          (string) An operationid to pass to z_getoperationstatus to get the result of the operation.\n"
+            "  \"opid\": xxx                         (string) An operationid to pass to z_getoperationstatus to get the result of the operation.\n"
```

Comment:
Should it be "xxx" as it is a string?

Reply:
We seem not to be consistent about this, and I could not find any examples of us using "xxx", so I am leaving this as is for the time being.